index.js

```diff
@@ -20,11 +20,12 @@ module.exports = declare((api, options) => {
   api.assertVersion('^7.0.0');
 
   const {
-    modules,
+    modules = 'auto',
     targets = buildTargets(options),
     removePropTypes,
     looseClasses = false,
     runtimeVersion,
+    useESModules = !modules,
```

Comment:
I think the name of this option is a little confusing since this only applies to runtime helpers. What do you think about `runtimeHelpersUseESModules`?